### PR TITLE
[Fix] Update slash command dispatch actions 

### PR DESCRIPTION
**Pull request by prateekdesai04**
*Description of changes:*
Urgent update of all slash-command-dispatch actions versions.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by yinweisu** [comment]
| Previous CI Run | Current CI Run |
| :---: | :---: |

**Comment by tonyhoo** [comment]
thanks for the PR. How can we validate this fix?

**Review by suzhoum — APPROVED** [review]
Can we link the ticket here as well?

**Comment by prateekdesai04** [comment]
> thanks for the PR. How can we validate this fix?

@tonyhoo we can try to see if we can reproduce this with the latest version

**Comment by prateekdesai04** [comment]
> Can we link the ticket here as well?

I don't think we can for security reasons

**Review by tonyhoo — APPROVED** [review]

**Review by Innixma — APPROVED** [review]
LGTM!

File: .github/workflows/benchmark-command.yml [hunk 1]
```diff
@@ -77,7 +77,7 @@ jobs:
 
       - name: Create comment
         if: (github.event_name == 'workflow_dispatch')
-        uses: peter-evans/create-or-update-comment@v1
```
Comment (tonyhoo) [inline .github/workflows/benchmark-command.yml:80]: Can we pin the major version so that we can get updates?

**Comment by yinweisu** [comment]
| Previous CI Run | Current CI Run |
| :---: | :---: |

**Pull request merged**